README.md

```diff
@@ -26,7 +33,11 @@ To use the downloader in a question-answer format, run it by executing the follo
 npx @adobe/reactor-downloader
 ```
 
-The tool will ask for any information necessary to download the repository.
+The tool will ask for any information necessary to download the repository.  If by chance you get an error when trying to run this command that is similar to `sh: reactor-downloader: command not found`, run the following command and try rerunning the script once done:
```

Comment:
let's stick with using `npx` to run reactor-downloader. that way users don't have to worry about which version of reactor-downloader they have installed.

also, once this is package is in an ok place, I'll publish it to the npm registry so we can change the command to:
`npx @tyssejc/reactor-downloader`. if users run `npx @adobe/reactor-downloader`, that will always run the latest version of the abandoned package, rather than this one